app/src/main/java/com/testtask/apptesttask/model/storage/Prefs.kt

```diff
@@ -0,0 +1,32 @@
+package com.testtask.apptesttask.model.storage
+
+import android.content.Context
+import com.google.gson.Gson
+import javax.inject.Inject
+
+class Prefs @Inject constructor(context: Context, val gson: Gson) {
+
+    private val PREF_TAG = "characters"
+    private val FAVORITE_CHARACTER = "favorite_characters"
+
+    private val sharedPreferences =
+            context.getSharedPreferences(PREF_TAG, Context.MODE_PRIVATE)
+
+    var favoritesCharacters: MutableList<Boolean>?
```

Comment:
Instead of this implementation, keep ` Map<Id, Character>` with only favorite objects.